_data/team/sebastian-funk.yml

```diff
@@ -1,9 +1,8 @@
 name: Sebastian Funk
 webpage: https://www.lshtm.ac.uk/aboutus/people/funk.sebastian
 github: sbfnk
+twitter: sbfnk
 orcid: 0000-0002-2842-3406
 current-member: true
 description: > 
-  Nam ut est ad modi illo aperiam. Tenetur enim sed voluptatibus repellat 
-  dolores et aut placeat. Inventore voluptate enim quam vel. Voluptatem est 
-  veniam eius sunt porro error.
+  My main interest is in how we can use computational models in combination with infectious disease data in order to better understand and predict infectious disease dynamics. This includes developing and evaluating methods for short-term forecasting, testing the predictive value of additional data sources on pathogen biology and human behaviour, and doing all of this in order to inform public health decision making. I strongly believe that science should done openly and thing there is great value in developing methods as tools that can be used by others.  
```

Comment:
Only small detail is that others used the 3rd person for their description. I don't know if we should try to use a homogeneous style here or not. Otherwise, LGTM so feel free to merge.